Makefile

```diff
@@ -380,16 +397,12 @@ release-publish: release-prereqs $(UPLOAD_DIR) helm-index
 	@echo ""
 	@echo "  https://github.com/projectcalico/calico/releases/tag/$(CALICO_VER)"
 	@echo ""
+	$(MAKE) helm-index
 
-## Updates helm-index with the new release chart
-helm-index: release-prereqs
-	rm -rf  charts
-	mkdir -p charts/$(CALICO_VER)/
-	cp $(RELEASE_HELM_CHART) charts/$(CALICO_VER)/
-	wget https://calico-public.s3.amazonaws.com/charts/index.yaml -O charts/index.yaml.bak
-	cd charts/ && helm repo index . --merge index.yaml.bak --url https://github.com/projectcalico/calico/releases/download/
-	aws --profile helm s3 cp index.yaml s3://calico-public/charts/ --acl public-read
-	rm -rf charts
+## Kicks semaphore job which syncs github released helm charts with helm index file
+.PHONY: helm-index
+helm-index:
+	SEMAPHORE_PROJECT_ID=30f84ab3-1ea9-4fb0-8459-e877491f3dea SEMAPHORE_WORKFLOW_BRANCH=master SEMAPHORE_WORKFLOW_FILE=../releases/calico/helmindex/update_helm.yml $(MAKE) semaphore-run-workflow
```

Comment:
Is the `SEMAPHORE_PROJECT_ID` always going to be the same? Wondering if we should keep this hardcoded in or if it might be better to do a check to ensure that it's populated and error out if it isn't so it becomes on the user to provide the ID if we expect it to ever differ.

Reply:
I wavered on this one, honestly.  It would only ever change if we deleted that project from semaphore and re-added it, which I think is unlikely, so hardcoding made sense.  The common Makefile has a section which defines all the project IDs for all our projects, but it does not include this one (the process project).